vogon/mysql_settings.py

```diff
@@ -125,3 +125,6 @@
 sys.path.append('/diging/wwwnginx/vogon')
 from secrets import ERATOSTHENES_TOKEN
 ERATOSTHENES_ENDPOINT = 'http://diging.asu.edu:9000/eratosthenes'
+
+CONCEPTPOWER_USERID = os.environ.get('ADMIN_ACCESS_KEY', None)
```

Comment:
These global names are much better. But why would we use different names for the environment variables used to control them?